### PR TITLE
Reorder offsets

### PR DESCRIPTION
**Pull request by aleix11alcacer**
This PR will allow Blosc to reorder the chunks of a super-chunk.

### CODE REVIEW

File: blosc/blosc2.h [hunk 1]
```diff
@@ -1081,6 +1081,15 @@ BLOSC_EXPORT int blosc2_schunk_get_cparams(blosc2_schunk *schunk, blosc2_cparams
  */
 BLOSC_EXPORT int blosc2_schunk_get_dparams(blosc2_schunk *schunk, blosc2_dparams **dparams);
 
+/**
+ * @brief Reorder the offsets of an existing super-chunk.
+ *
+ * @param schunk The super-chunk from where rewrite the offsets.
```
Comment (FrancescAlted) [inline blosc/blosc2.h:1087]: Suggestion: "The super-chunk whose chunks are to be reordered."

File: blosc/blosc2.h [hunk 2]
```diff
@@ -1081,6 +1081,15 @@ BLOSC_EXPORT int blosc2_schunk_get_cparams(blosc2_schunk *schunk, blosc2_cparams
...
+ * @param offsets_order The new order for the offsets.
```
Comment (FrancescAlted) [inline blosc/blosc2.h:1088]: Suggestion: "The new order of the offsets."

File: tests/test_reorder_offsets.c [hunk 3]
```diff
@@ -0,0 +1,106 @@
+/*
+  Copyright (C) 2019  Francesc Alted
+  http://blosc.org
+  License: BSD (see LICENSE.txt)
+
+  Creation date: 2019-08-06
```
Comment (FrancescAlted) [inline tests/test_reorder_offsets.c:6]: Updated date.

File: tests/test_reorder_offsets.c [hunk 4]
```diff
@@ -0,0 +1,106 @@
+/*
+  Copyright (C) 2019  Francesc Alted
```
Comment (FrancescAlted) [inline tests/test_reorder_offsets.c:2]: Copyright should be "The Blosc Developers" or "The Blosc Development Team".  Check with the LICENSE file.

**Review by FrancescAlted — CHANGES_REQUESTED** [review]

File: tests/test_reorder_offsets.c (hunk 4 above)
Comment (aleix11alcacer) [reply tests/test_reorder_offsets.c]: I changed it. Ok now?

File: blosc/blosc2.h [hunk 5]
```diff
@@ -1081,6 +1081,16 @@ BLOSC_EXPORT int blosc2_schunk_get_cparams(blosc2_schunk *schunk, blosc2_cparams
  */
 BLOSC_EXPORT int blosc2_schunk_get_dparams(blosc2_schunk *schunk, blosc2_dparams **dparams);
 
+/**
+ * @brief Reorder the offsets of an existing super-chunk.
```
Comment (FrancescAlted) [inline blosc/blosc2.h:1085]: Suggestion: "offsets" -> "chunk offsets" (better be explicit)

File: blosc/blosc2.h [hunk 6]
```diff
@@ -1081,6 +1081,16 @@ BLOSC_EXPORT int blosc2_schunk_get_cparams(blosc2_schunk *schunk, blosc2_cparams
...
+ *
+ * @param schunk The super-chunk whose chunks are to be reordered.
```
Comment (FrancescAlted) [inline blosc/blosc2.h:1087]: Suggestion: "chunks" -> "chunk offsets" (better be explicit)

File: blosc/blosc2.h [hunk 7]
```diff
@@ -1081,6 +1081,16 @@ BLOSC_EXPORT int blosc2_schunk_get_cparams(blosc2_schunk *schunk, blosc2_cparams
...
+ * @param offsets_order The new order of the offsets.
```
Comment (FrancescAlted) [inline blosc/blosc2.h:1088]: Suggestion: "offsets" -> "chunk offsets" (better be explicit)

**Review by FrancescAlted — APPROVED** [review]
Good to merge!

**Pull request merged**